packages/gatsby/src/commands/build-javascript.ts

```diff
@@ -1,14 +1,16 @@
-import webpack from "webpack"
+import { Span } from "opentracing"
+import webpack, { Stats } from "webpack"
+
 import webpackConfig from "../utils/webpack.config"
+
 import { IProgram } from "./types"
-import { Span } from "opentracing"
 
 import { reportWebpackWarnings } from "../utils/webpack-error-utils"
 
 export const buildProductionBundle = async (
   program: IProgram,
   parentSpan: Span
-): Promise<webpack.Stats> => {
+): Promise<Stats> => {
```

Comment:
I actually prefer the namespacing, makes it clearer what Stats is for. Can you revert this line, happy to take the import clean up though :)